pom.xml

```diff
@@ -2541,6 +2541,10 @@
         <activemq.broker.version>5.15.2</activemq.broker.version>
         <net.minidev.accessors-smart.version>1.2</net.minidev.accessors-smart.version>
         <org.ow2.asm.asm.version>5.2</org.ow2.asm.asm.version>
+
+        <!--SAP import version. This should be changed immediately after each release of carbon-mediation to the
+        latest released version of mediation.-->
+        <org.wso2.carbon.transports.sap.import.version>4.6.129-SNAPSHOT</org.wso2.carbon.transports.sap.import.version>
```

Comment:
Can't we use the `${carbon.mediation.version}` here?

Reply:
SAP jco jar are required to build the component. And not everybody who's building the mediation would have these jars. So to avoid failure of the local build, this should ideally be latest release version of carbon-mediation.

Reply:
can we use build-helper-maven-plugin with <goal>released-version</goal> and get the releasedVersion.version instea of manually updating

Reply:
@Nirothipan will check on it.

Reply:
Corrected it. Thank you very much for the input. :)